product_sticker/models/product_sticker.py

```diff
@@ -0,0 +1,176 @@
+from odoo import api, fields, models, tools
+from odoo.osv import expression
+
+
+class ProductSticker(models.Model):
+    _name = "product.sticker"
+    _description = "Product Sticker"
+    _inherit = ["image.mixin"]
+    _order = "sequence, id"
+
+    company_id = fields.Many2one(
+        comodel_name="res.company",
+        string="Company",
+        default=lambda s: s.env.company,
+    )
+    sequence = fields.Integer(default=10, index=True)
+    name = fields.Char(required=True, translate=True)
+    image_1920 = fields.Image(required=True)
+    product_category_id = fields.Many2one(
+        comodel_name="product.category",
+        string="Category",
+        ondelete="cascade",
+    )
+    product_attribute_id = fields.Many2one(
+        comodel_name="product.attribute",
+        string="Attribute",
+        ondelete="cascade",
+    )
+    product_attribute_value_id = fields.Many2one(
+        comodel_name="product.attribute.value",
+        string="Attribute Value",
+        ondelete="cascade",
+        domain="[('attribute_id', '=', product_attribute_id)]",
+    )
+    can_image_128_be_zoomed = fields.Boolean(
+        string="Can Image 128 be zoomed",
+        compute="_compute_can_image_128_be_zoomed",
+        store=True,
+    )
+    show_sticker_name = fields.Boolean(
+        string="Display Name with the sticker",
+        help="If checked, the name will be displayed with the sticker",
+    )
```

Comment:
Just to make upper case more consistent:

```suggestion
    product_attribute_value_id = fields.Many2one(
        comodel_name="product.attribute.value",
        string="Attribute value",
        ondelete="cascade",
        domain="[('attribute_id', '=', product_attribute_id)]",
    )
    can_image_128_be_zoomed = fields.Boolean(
        string="Allow zooming in 128px image",
        compute="_compute_can_image_128_be_zoomed",
        store=True,
    )
    show_sticker_name = fields.Boolean(
        string="Display name with the sticker",
        help="If checked, the name will be displayed with the sticker",
    )
```